src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  margin: 0;
```

Comment:
This only resets margin for the body, not for all elements.

Reply:
As Radek mention reset margin only for the body and we are good to go